nncf/quantization/quantize_model.py

```diff
@@ -81,6 +81,10 @@ def quantize(
     :return: The quantized model.
     :rtype: TModel
     """
+
+    if subset_size < 1:
```

Comment:
@alexsu52 is it correct place to add checks of parameters?

Can it be done all checks here? 
```
if not self.dataset.get_length():
    raise RuntimeError("Calibration dataset must not be empty.")
```

Reply:
`get_length()` returns None for datasets that don't have `__len__` implemented, so we can't check it here for sure

Reply:
Sad 😞

Reply:
This is correct place to check value of subset_size parameter.